threshold_tools_application_examples.ipynb

```diff
@@ -121,9 +124,10 @@
     "For this section, please select:\n",
     "- \"daily\" timescale,\n",
     "- \"2m Air Temperature\",\n",
+    "- \"degC\" for units,\n",
     "- \"9 km\" resolution,\n",
     "- \"SSP3-7.0 -- Business as Usual\" scenario with \"Append historical\",\n",
-    "- and \"Ca counties\" area subset with \"Scramento County\" cached area."
+    "- and \"CA counties\" area subset with \"Scramento County\" cached area."
```

Comment:
Typo here, "Scramento" should be "Sacramento" ;)

Reply:
Lol - great catch. Fixing now